### PR TITLE
Updated the confusing element from README

### PR DESCRIPTION
**Pull request by Kaushl2208**
#558

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ You can specify either a package (+ version) or a local directory to analyze:
 
 ```
 Usage: pana [<options>] <published package name> [<version>]
-       pana [<options>] --source path <local directory>
+       pana [<options>] --source path/to/local/directory
```
Comment (isoos) [inline README.md]: I think this is not correct. Source has only two values allowed:
https://github.com/dart-lang/pana/blob/master/bin/pana.dart#L30

The current version of the readme is correct in that `--source path` is the verbatim command followed by a path to local package dir.

Comment (Kaushl2208) [reply README.md]: Hey @isoos, I see what you are trying to say but in original README the `--source path` path was quite confusing that's why I have mentioned to/local/directory(it gives a sort of explanation).

Comment (Kaushl2208) [reply README.md]: I see, What I have done wrong here. It should look like `--source path <to/local/directory>`.

Comment (isoos) [reply README.md]: Yes, `--source path <to/local/directory>` is correct.

**Comment by Kaushl2208** [comment]
Hey @isoos, Updated the Readme as discussed. I don't know why checks are failing. Can you help me with that.

**Comment by isoos** [comment]
> I don't know why checks are failing. Can you help me with that.

There were CI issues that are now fixed on the head of the master branch. Assuming you are using the usual git setup, the following rebase should work:

```
git checkout master
git pull upstream master
git checkout Doc_Update
git rebase master
git push -f origin Doc_Update
```

**Comment by Kaushl2208** [comment]
Thank you for your help. @isoos

**Comment by sigurdm** [comment]
It seems this needs an update of test/bin_pana_test.dart to expect the modified --help text

**Comment by mit-mit** [comment]
Closing as stale

**Pull request closed without merging**